oktapam/resource_team_settings.go

```diff
@@ -0,0 +1,208 @@
+		for key, value := range settings.ToResourceMap() {
+			err = d.Set(key, value)
+			if err != nil {
+				return diag.FromErr(err)
+			}
+		}
+	} else {
+		return diag.Errorf("Team settings does not exist for the team %s", c.Team)
+	}
+
+	return diags
+}
+
+func resourceTeamSettingsUpdate(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
+	c := m.(client.OktaPAMClient)
+
+	//if d.HasChanges(attributes.ReactivateUsersViaIDP, attributes.ApproveDeviceWithoutInteraction, attributes.PostDeviceEnrollmentURL, attributes.PostLoginURL,
+	//	attributes.PostLogoutURL, attributes.UserProvisioningExactUserName, attributes.ClientSessionDuration, attributes.WebSessionDuration, attributes.IncludeUserSID) {
+	//
+	//	//Build API Client Request Object
+	//	teamSettingsRequest := client.TeamSettings{
+	//		ReactivateUsersViaIDP:           getBoolPtr(attributes.ReactivateUsersViaIDP, d, false),
+	//		ApproveDeviceWithoutInteraction: getBoolPtr(attributes.ApproveDeviceWithoutInteraction, d, false),
+	//		PostDeviceEnrollmentURL:         getStringPtr(attributes.PostDeviceEnrollmentURL, d, false),
+	//		PostLogoutURL:                   getStringPtr(attributes.PostLogoutURL, d, false),
+	//		PostLoginURL:                    getStringPtr(attributes.PostLoginURL, d, false),
+	//		UserProvisioningExactUserName:   getBoolPtr(attributes.UserProvisioningExactUserName, d, false),
+	//		ClientSessionDuration:           getIntPtr(attributes.ClientSessionDuration, d, false),
+	//		WebSessionDuration:              getIntPtr(attributes.WebSessionDuration, d, false),
+	//		IncludeUserSID:                  getStringPtr(attributes.IncludeUserSID, d, false),
+	//	}
+	//
+	//	err := c.UpdateTeamSettings(ctx, teamSettingsRequest)
+	//	if err != nil {
+	//		return diag.FromErr(err)
+	//	}
+	//}
+
+	changed := false
+	updates := make(map[string]any)
+
+	changeableAttributes := []string{
+		attributes.ReactivateUsersViaIDP,
+		attributes.ApproveDeviceWithoutInteraction,
+		attributes.PostDeviceEnrollmentURL,
+		attributes.PostLoginURL,
+		attributes.PostLogoutURL,
+		attributes.UserProvisioningExactUserName,
+		attributes.ClientSessionDuration,
+		attributes.WebSessionDuration,
+		attributes.IncludeUserSID,
+	}
+
+	for _, attribute := range changeableAttributes {
+		if d.HasChange(attribute) {
+			updates[attribute] = d.Get(attribute)
+			changed = true
+		}
+	}
+
+	if changed {
+		err := c.UpdateTeamSettings(ctx, updates)
+		if err != nil {
+			return diag.FromErr(err)
+		}
+	}
+
+	return resourceTeamSettingsRead(ctx, d, m)
+}
+
+func resourceTeamSettingsDelete(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
+	var diags diag.Diagnostics
+	diag := diag.Diagnostic{
+		Severity: diag.Warning,
+		Summary:  "Can't delete team settings resource.",
+		Detail: fmt.Sprintf("Team settings resource does not support delete operation. If you don't want to see this warning, plese remove this resource from the terraform state manually."),
```

Comment:
nit: s/plese/please

Reply:
nit: If you don't want to manage team settings via terraform, then remove it from the terraform state manually using "state rm" command.